### PR TITLE
Setup wizard refactor

### PR DESCRIPTION
**Pull request by pratu16x7**
Depends on frappe/frappe#3548

### CODE REVIEW

File: erpnext/public/js/setup_wizard.js [hunk 1]
```diff
@@ -7,459 +7,391 @@ frappe.pages['setup-wizard'].on_page_load = function(wrapper) {
...
-					slide.fields = slide.fields.concat([
-						{fieldtype:"Section Break"},
-						{fieldtype:"Data", fieldname:"supplier_" + i, label:__("Supplier")+" " + i,
-							placeholder:__("Supplier Name")},
-						{fieldtype:"Column Break"},
-						{fieldtype:"Data", fieldname:"supplier_contact_" + i,
-							label:__("Contact Name") + " " + i, placeholder:__("Contact Name")},
-					])
+				var year_start_date = current_year + "-" + fy[0];
+				if (year_start_date > get_today()) {
```
Comment (mbauskar) [inline erpnext/public/js/setup_wizard.js]: `get_today()` deprecated use `frappe.datetime.get_today()`

**Review by mbauskar — CHANGES_REQUESTED** [review]

**Comment by mbauskar** [comment]
push the changes and reopen the PR

**Comment by pratu16x7** [comment]
Frappe header:

screen shot 2017-06-28 at 9 47 57 am

**Comment by mbauskar** [comment]
@pratu16x7,

Need to set the correct fiscal year for country ..

screen shot 2017-07-03 at 12 37 55 pm

**Pull request merged**